src/admin-integration-email-list/admin-integration-email-list.html

```diff
@@ -0,0 +1,31 @@
+<h2>{{'adminIntegrationEmailList.header' | message}}</h2>
+<section class="openlmis-table-container">
+    <button id="add-email" class="add" ui-sref="openlmis.administration.adminIntegrationEmailList.add">
+        {{'adminIntegrationEmailList.addEmail' | message}}
```

Comment:
same as above, if the text is not changed or being affected by ng-repeat, use one time binds